Lib/exception_group.py

```diff
@@ -0,0 +1,215 @@
+
+import sys
+
+
+class TracebackGroup:
+    def __init__(self, excs):
+        self.tb_next_map = {} # exception id to tb
+        for e in excs:
+            if isinstance(e, ExceptionGroup):
+                for e_ in e.excs:
+                    self.tb_next_map[id(e_)] = e_.__traceback__
+            else:
+                if e.__traceback__:
+                    self.tb_next_map[id(e)] = e.__traceback__.tb_next
+                else:
+                    self.tb_next_map[id(e)] = None
+
+class ExceptionGroup(BaseException):
+
+    def __init__(self, excs, *, tb=None):
+        """ Construct a new ExceptionGroup
```

Comment:
What would you think of adding a ‘message’ parameter here? Yury brought this up in the discussion.

Reply:
Hm, I had assumed the signature would have to be
```
def __init__(self, msg, excs, *, tb=None):
```
so that `msg` ends up in `self.args[0]`, where code dealing with exceptions in general expects a message.